src/NzbDrone.Core/Notifications/Join/JoinProxy.cs

```diff
@@ -94,8 +94,8 @@ private void SendNotification(string title, string message, HttpMethod method, J
             var request = requestBuilder.AddQueryParam("apikey", settings.ApiKey)
                           .AddQueryParam("title", title)
                           .AddQueryParam("text", message)
-                          .AddQueryParam("icon", "https://cdn.rawgit.com/Sonarr/Sonarr/phantom-develop/Logo/256.png") // Use the Radarr logo.
-                          .AddQueryParam("smallicon", "https://cdn.rawgit.com/Sonarr/Sonarr/phantom-develop/Logo/96-Outline-White.png") // 96x96px with outline at 88x88px on a transparent background.
+                          .AddQueryParam("icon", "https://cdn.rawgit.com/Sonarr/Sonarr/develop/Logo/256.png") // Use the Sonarr logo.
```

Comment:
maybe we should update this to `/main/` instead of `/develop/`

Reply:
Main may make sense, as in the case of a logo change it'll be right for most of the user base all the time.